docs/install/local/README.md

```diff
@@ -233,6 +233,25 @@ you can use a pre-built docker image that provides everything needed.
 
     This will map the `1883`/`1884` ports to the host machine so they can be accessed outside of the container. If you already have an MQTT broker running on port 1883, then you'll need to modify the `-p` options to use a different set of ports. For example: `-p 9883:1883 -p 9884:1884`.
 
+## File Server
+
+By default the FlowFuse File Server component is disabled as it is a licensed feature. If you provide a license you can start the File Server with the following command:
+
+```bash
+sudo service flowforge-file start
+```
+
+You can then uncomment the following section in the `/opt/flowforge/etc/flowforge.yml` file
+
+```yaml
+#################################################
+# File Server config                            #
+#################################################
+
+fileStore:
+  url: http://localhost:3001
+
+```
```

Comment:
Ben, should these not be reversed? (i.e. setting YAML BEFORE launching service)


**Unfortunately, due to multiple code fences, I cannot do this as a suggestion (it messes up the code suggestion)**

Here is my proposal:

<pre>
By default the FlowFuse File Server component is disabled as it is a licensed feature.
If you provide a license you can start the File Server with the following instructions:

Uncomment the `fileStore` keys in the `/opt/flowforge/etc/flowforge.yml` file:

```yaml
#################################################
# File Server config                            #
#################################################

fileStore:
  url: http://localhost:3001
```

Then you can start the File Server with the command:

```bash
sudo service flowforge-file start
```
</pre>

Reply:
NO!

Reply:
Because you have to start the file-server before you can try and use it

Reply:
Ah, soory, this wasnt clear at all.

Now you have emphatically stated NO I thought about it (and read it again) and realised that the yml file you are talking about is NOT for THIS application but actually for the FlowFuse application.  That makes sense now, but I am suspect that will not be clear to others either.

Question: Should we also state that FlowFuse application will need a restart after modifying `/opt/flowforge/etc/flowforge.yml`?  As it stands, this implies simply updating the yml is enough.


> Because you have to start the file-server before you can try and use it

Should that be included in docs?


> If you provide a license you can start the File Server with the following command

That reads like "If i provide a license to the File Server it will work". Should we not be clear about (or link to) the licensing? 

---

I got tripped up, imagine those reading this for the first time 🤷‍♂️

Reply:
You already have to restart the forge app after applying a license to enable the features so it's covered.

Reply:
ok